packages/terra-functional-testing/src/config/utils/getMaxInstances.js

```diff
@@ -0,0 +1,7 @@
+module.exports = () => {
+  const args = process.argv.slice(2);
```

Comment:
why do we need to slice 2 here ..?

Reply:
Not required removed [bb1cedd](https://github.com/cerner/terra-toolkit/pull/837/commits/bb1cedd1374af23a3587607da5145008cb3d688b)